src/algebraic_topology/simplicial_type.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2021 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Scott Morrison
+-/
+import algebraic_topology.simplicial_object
+import category_theory.yoneda
+
+/-!
+A simplicial type (usually called a simplicial set, when using set theoretic foundations)
```

Comment:
I think this term "simplicial type" is not good, as I will explain in the main comment thread.